data/bello-olalekan.json

```diff
@@ -2,13 +2,19 @@
   "name": "Bello Olalekan",
   "displayStatsPublic": true,
   "type": "personal",
-  "bio": "SRE | DevOps Engineer **. Combat Sports fan",
+  "bio": "SRE | DevOps Engineer. Cloud Native and Open Source Enthusiast",
   "avatar": "https://github.com/bello-olalekan.png",
   "links": [
+     {
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/bello-olalekan",
```

Comment:
fix the LinkedIn link, it should be `https://www.linkedin.com/in/bello-olalekan`